cactusbot/commands/magic/cactus.py

```diff
@@ -0,0 +1,38 @@
+"""Cactus Command."""
+
+
+from . import Command
+from ...packets import MessagePacket
+
+class Cactus(Command):
+    """Ouch! That's pokey!"""
+
+    COMMAND = "cactus"
+
+    @Command.subcommand()
+    async def cactus(self):
+        return MessagePacket(
+            ("text", "Ohai! I'm CactusBot! "),
+            ("emoji", ":cactus:", ":cactus:")
+        )
+
+    @Command.subcommand
+    async def docs(self):
+        return MessagePacket(
+            ("text", "Check out my documentation at "),
+            ("link", "https://cactusbot.rtfd.org", "cactusbot.rtfd.org"),
+            ("text", ".")
+        )
+
+    @Command.subcommand
+    async def help(self):
+        return "Try our docs (!cactus docs). If that doesn't help, tweet at us (!cactus twitter)!"
+    
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/cactusbot/commands/magic/cactus.py
+++ b/cactusbot/commands/magic/cactus.py
@@ -27,7 +27,7 @@
     @Command.subcommand
     async def help(self):
         return "Try our docs (!cactus docs). If that doesn't help, tweet at us (!cactus twitter)!"
-    
+
     @Command.subcommand()
     async def twitter(self):
         return MessagePacket(
```